ie_serving/main.py

```diff
@@ -64,22 +65,46 @@ def parse_config(args):
     check_config_structure(configs=configs)
     models = {}
     for config in configs['model_config_list']:
-        if 'batch_size' in config['config'].keys():
-            batch_size = config['config']['batch_size']
-        else:
-            batch_size = None
-        model = ModelBuilder.build(model_name=config['config']['name'],
-                                   model_directory=config
-                                   ['config']['base_path'],
-                                   batch_size=batch_size)
-        models[config['config']['name']] = model
+        try:
+            batch_size = config['config'].get('batch_size', None)
+            model_ver_policy = config['config'].get(
+                'model_version_policy', None)
+            model = ModelBuilder.build(model_name=config['config']['name'],
+                                       model_directory=config['config'][
+                                           'base_path'],
+                                       batch_size=batch_size,
+                                       model_version_policy=model_ver_policy)
+            models[config['config']['name']] = model
+        except ValidationError as e_val:
+            logger.warning("Problem with model_version_policy in {} model. "

```

Comment:
`Problem with model_version_policy in {} model. `
maybe `model version policy for model {} is Invalid`?